internal/storage/bucket_handle.go

```diff
@@ -483,6 +483,7 @@ func (b *bucketHandle) DeleteFolder(ctx context.Context, folderName string) (err
 		if !errors.As(err, &notfound) {
 			return err
 		}
+		logger.Infof("Object in DeleteObject not found for delete folder: %v", err)
```

Comment:
change level to trace

Reply:
any error should be ignored here, please ad a comment on why its ignored

Reply:
Done